test/foundry/dapp/Application.t.sol

```diff
@@ -541,6 +593,23 @@ contract ApplicationTest is ERC165Test {
         _finishInput();
         _finishEpoch();
 
+        _nameOutput(
+            "ERC20DelegateCallVoucher",
+            _addOutput(
+                _encodeDelegateCallVoucher(
+                    address(_contractERC20Transfer),
+                    abi.encodeWithSignature(
```

Comment:
Prefer `abi.encodeCall` over `abi.encodeWithSignature`, as the former allows the compiler to make type checks on the arguments of the call.

Reply:
Yes `abi.encodeCall` is preferred. But we need an interface to be used by `encodeCall`, right?

Reply:
Yes. Should use `abi.encodeCall` instead.